app/controllers/file_controller.rb

```diff
@@ -17,18 +17,19 @@ def show
 
   private
 
+  def allowed_params
+    params.permit(:id, :file_name)
+  end
+  alias stacks_file_params allowed_params
+
```

Comment:
My thought was that it was just a coincidence that the stacks_file_params and the allowed_params were the same.  They might change independently.  Perhaps this is a premature optimization, or perhaps it's a useful alias showing the differing functions of the two calls.